backend/index.js

```diff
@@ -0,0 +1,32 @@
+import fs from "fs";
+import express from "express";
+
+const app = express();
+
+let pageVisitedTimesSession = 0;
+let pageVisitedTimesTotal = 0;
+
+try {
+  if (fs.existsSync("temp/my-counter-visited-times.txt")) {
+    (pageVisitedTimesTotal = Number(fs.readFileSync("temp/my-counter-visited-times.txt", "utf8")))
+  }
+} catch (e) {
+  console.log(e);
+}
+
+app.get("/", (req, res) => {
+  ++pageVisitedTimesSession;
+
+  res.status(200).send(
+    `This page was visited ${++pageVisitedTimesTotal
```

Comment:
Never do `++something` inside in places like this or when passing it to a function.
Basicly the only place when ++something should be written is on it's own separate line.
Why? Because when someone else is reading your code it's much harder to keep track of value changes if they are not on separate lines.